CourseGrab/ViewControllers/Search/SearchTableViewController.swift

```diff
@@ -74,11 +66,17 @@ extension SearchTableViewController {
         cell.configure(for: sections[indexPath.row])
         return cell
     }
-
+    
```

Comment:
remove whitespace

Reply:
In the file there is only one line of whitespace, that is protocol between functions right? @kevinchan159